src/content/docs/apm/transactions/transaction-traces/transaction-traces-database-queries-page.mdx

```diff
@@ -31,69 +33,30 @@ If database query data is associated with your [selected transaction trace](/doc
 If you expect to see database data for a transaction trace but don't see it, you may need to change the [Database query settings](#settings).
 
 <img
-  title="APM-transaction-trace-database-queries.png"
-  alt="APM - Transaction trace - Database queries tab"
-  src={apmTransactionTraceDbQuery}
+    title="transaction trace database queries tab"
+    alt="A screenshot depicting the transaction trace database queries tab in the UI."
+    src={apmTransactionTraceDatabaseQueriesTab}
 />
-
 <figcaption>
   Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM & services > (select an app) > Monitor > Transactions > (select a transaction trace) > Database queries**.
 </figcaption>
 
-## Use Database queries [#use]
-
-Use the **Database queries** page to analyze and troubleshoot a slow transaction. For example:
-
-1. You notice on the [APM **Overview** page](/docs/apm/applications-menu/monitoring/apm-overview-page) that a transaction is abnormally slow.
-2. You select a [transaction trace](/docs/apm/transactions/transaction-traces/transaction-traces) to get more details about that transaction.
-3. If the trace has associated database data, you select the **Database queries** page to examine the slow queries in the trace, and use them as the basis to [troubleshoot and improve your app's performance](#troubleshooting).
+## Use database queries [#use]
 
-Other functions include:
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "200px" }}>
-        If you want to...
-      </th>
+Here are situations where you can use the **Database queries** page to analyze and troubleshoot a slow transaction:
 
-      <th>
-        Do this...
-      </th>
-    </tr>
-  </thead>
+* You notice on the [APM **Overview** page](/docs/apm/applications-menu/monitoring/apm-overview-page) that a transaction is abnormally slow.
+* You select a [transaction trace](/docs/apm/transactions/transaction-traces/transaction-traces) to get more details about that transaction.
+* You select the **Database queries** page to examine the slow queries in the trace, and use them as the basis to [troubleshoot and improve your app's performance](#troubleshooting).
 
-  <tbody>
-    <tr>
-      <td>
-        Hide fast calls
```

Comment:
@homelessbirds it looks like you dropped the information in the table about hiding fast calls. Is that no longer relevant? Also, why not just make the "View stack traces" from that old table a final bullet and insert the image indented under it? The way it appears now is with an orphaned h3 with no siblings.

Reply:
@Marty42Martinez - can you confirm that I can delete this section @Marty42Martinez ?

Reply:
Resolved this in a DM with Marty.

Reply:
Thanks @rhetoric101 . This is done.